File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.cs
public class Solution {
    public int[] SearchRange(int[] nums, int target) {
        int start = -1;
        int end = -1;

        for (int i = 0; i < nums.Length; i++) {
            if (nums[i] == target) {
                start = i;
                break;
            }
        }

        for (int i = nums.Length - 1; i >= 0; i--) {
            if (nums[i] == target) {
                end = i;
                break;
            }
        }

        return new int[] { start, end };
    }
}

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        start = -1
        end = -1
        for i in range(len(nums)):
            if nums[i] == target:
                start = i
                break
        for i in range(len(nums)-1, -1, -1):
            if nums[i] == target:
                end = i
                break
        return [start, end]